homeassistant/components/cast/strings.json

```diff
@@ -21,12 +21,19 @@
   },
   "options": {
     "step": {
-      "options": {
-        "description": "Please enter the Google Cast configuration.",
+      "basic_options": {
+        "title": "Google Cast configuration",
+        "description": "### Known Hosts \n A comma-separated list of hostnames or IP-addresses of cast devices, use if mDNS discovery is not working.",
```

Comment:
We generally have not used markdown in descriptions. I don't think that we should start doing it without a styleguide for it.

Reply:
Right, OK.
How do we get started to compile a styleguide?

Reply:
If you need such help texts, we shouldn't abuse description but allow adding "help text" for the inputs.

Reply:
(which is not a feature that we support)

Reply:
Alright, the strings are shorter and all markdown removed.